docs/pbc/all/discount-management/import-and-export-data/file-details-discount-store.csv.md

```diff
@@ -29,8 +29,8 @@ data:import:discount-store
 
 | PARAMETER | REQUIRED | TYPE | REQUIREMENTS OR COMMENTS | DESCRIPTION |
 | --- | --- | --- | --- | --- |
-| discount_key | &check; | String |`discount_key` must be included in the `discount.csv` file. |  |
-| store_name | &check; | String |  | Name of the store to which the discount applies. |
+| discount_key | &check; | String |A `discount_key` must be included in the `discount.csv` file. |  |
```

Comment:
`discount_key` is a parameter name, so we shouldn't use the article in this case. So, it should be whether `discount_key` or the `discount_key` parameter.
```suggestion
| discount_key | &check; | String |`discount_key` must be included in the `discount.csv` file. |  |
```